auth0/src/main/java/com/auth0/android/authentication/request/SignUpRequest.java

```diff
@@ -118,6 +118,13 @@ public SignUpRequest setConnection(String connection) {
         return this;
     }
 
+    @Override
+    public SignUpRequest setRealm(String realm) {
+        signUpRequest.setConnection(realm);
+        authenticationRequest.setRealm(realm);
```

Comment:
And for connection? also if we are not using the proper grant (e.g. when working with the legacy endpoints) what will happen?. If we quietly ignore them we should at least issue a warning in the logs